package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "next dev -p $CONNECT_ACCOUNT_PORT",
     "build": "next build",
     "lint": "eslint \"{src,pages}/**/*.{js,ts,jsx,tsx}\"",
-    "start": "next start -p $CONNECT_ACCOUNT_PORT"
+    "start": "next start -p $CONNECT_ACCOUNT_PORT",
```

Comment:
I assume this is ill-advised if we want to deploy with vercel

Reply:
Checking the vercel deploy config file doc atm https://vercel.com/docs/configuration#introduction/configuration-reference, not sure how it's handled (probably injecting env vars).

Reply:
So the solution would be to have something like `"start": "next start -p $CONNECT_ACCOUNT_BUILD_PORT",`, and set up the env on the vercel console when we'll deploy, right ?

Reply:
c8257f4